packages/redis_client/lib/src/redis_client.dart

```diff
@@ -239,6 +245,64 @@ class RedisClient {
   }
 }
 
+/// {@template redis_json}
+/// An object that supports executing commands provided by the RedisJSON module.
+/// https://redis.io/docs/data-types/json/
+/// {@endtemplate}
```

Comment:
```suggestion
/// {@template redis_json}
/// An object that adds support for getting and setting JSON values. Backed by the RedisJSON module.
/// https://redis.io/docs/data-types/json/
/// {@endtemplate}
```

Reply:
haha thanks this was quite a clunky description 🤣

Reply:
Haha np, I think it was a fine description, just could use a bit "why should I use this" info :)